scripts/run-codecoverage.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env sh
+################################################################################
+#                              run-codecoverage                                #
+#                                                                              #
+# This script is used to run prepare-code-coverage-artifact.py python file to  #
+# to generate code coverage report for apollo tests.                           #
+#                                                                              #
+# Help:                                                                        #
+# # If do not want to preserve raw profile data files with the coverage report.#
+#   python3 $dir_path$py_file $raw_file_path_arg                               #
+#                                                                              #
+# # If do not want to preserve raw profile data files with the coverage report.#
```

Comment:
Are both these comments in Help to not preserve profile data. If not please modify comment appropriately.

Reply:
Comments have been fixed.